core/chains/solana/soltxm/txm_test.go

```diff
@@ -108,8 +111,8 @@ func TestTxm_Integration(t *testing.T) {
 	}
 
 	// check to make sure all txs are closed out from inflight list (longest should last MaxConfirmTimeout)
-	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
-	defer cancel()
+	ctx, cancel := context.WithCancel(ctx)
```

Comment:
:crossed_fingers: This may be enough to relieve the pressure - the test was quitting at 30s but the package has 2m.